src/NewCommand.php

```diff
@@ -288,7 +288,7 @@ protected function pushToGitHub(string $name, string $directory, InputInterface
         $branch = $input->getOption('branch') ?: $this->defaultBranch();
 
         $commands = [
-            "gh repo create {$name} -y {$flags}",
+            "gh repo create {$name} --source=. {$flags}",
```

Comment:
Why did you added this?

Reply:
To maintain current functionality, the installer can't push the git log if there is no origin remote:
> For example, running gh repo create myrepo --public in a local repository will no longer add a remote. It assumes you just want to create a new repository on GitHub. To create a remote repo from your local git repo, use gh repo create myrepo --public --source=. instead